services/QuillLMS/app/workers/copy_single_concept_result_worker.rb

```diff
@@ -16,7 +16,7 @@ def perform(old_concept_result_id)
 
     ActiveRecord::Base.transaction do
       directions = ConceptResultDirections.find_or_create_by(text: directions)&.id if directions
-      instructions ConceptResultInstructions.find_or_create_by(text: instructions)&.id if instructions
+      instructions = ConceptResultInstructions.find_or_create_by(text: instructions)&.id if instructions
```

Comment:
I was wondering the same thing.  I guess maybe it's the Ruby ability to parse function arguments without parens?  I think it parses that as `instructions(ConceptResultInstructions.find_or_create_by(text: instructions)&.id) if instructions`, but there is no function `instructions`...  Though, again, the parser should probably have noticed?

Reply:
Good point on the parentheses.  Since the ruby type checking happens at runtime instead of compile time, the parser doesn't know that instructions isn't a function and therefore allows the possibility that it's a function call.